src/easydmp/dmpt/models.py

```diff
@@ -59,6 +59,7 @@
     'multinamedurlonetext',
     'multidmptypedreasononetext',
     'multirdacostonetext',
+    'storageforecast'
```

Comment:
This should probably be "multistorageforecastonetext" which I suspect is longer than the input_type is allowed to be. Luckily, using a data migration to fix this later is easy.

.. I think the "onetext" should be a parameter on the field anyway. It is whether the answer is munged into a sentence in the html export. If there are more than three, a list would be better and we might also say a sentence is only used if there are less than three.